### PR TITLE
chore(starters): add gatsby-starter-service-relief

### PR DESCRIPTION
**Pull request by boborchard**
## Description

Adds the Gatsby Starter for the Service Relief Project to the starter library.

Documentation on how to get started using the starter is available here: https://www.servicerelief.us/start/

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -5716,3 +5716,18 @@
     - SEO optimized
     - Robots.txt
     - OpenGraph structured data
+- url: https://seattleservicerelief.com/
+  repo: https://github.com/service-relief/gatsby-starter-service-relief
```
Comment (laurieontech) [inline docs/starters.yml]:
```suggestion
  source_url: https://github.com/service-relief/gatsby-starter-service-relief
```

File: docs/starters.yml [hunk 2]
```diff
@@ -5716,3 +5716,18 @@
...
+  description: Localized index of resources for your city.
+  tags:
+    - Airtable
+    - Netlify
+    - SEO
+    - Styling:Tailwind
+    - Lightweight
```
Comment (laurieontech) [inline docs/starters.yml]:
```suggestion
```
Not a valid category.

Comment (boborchard) [reply docs/starters.yml]: 👍

**Comment by laurieontech** [comment]
Thanks Bob!

**Pull request merged**